wolfcrypt/src/port/nxp/se050_port.c

```diff
@@ -501,6 +501,72 @@ static sss_algorithm_t se050_map_hash_alg(int hashLen)
     return algorithm;
 }
 
+int se050_ecc_insert_private_key(int keyId, const byte* eccDer,
```

Comment:
I would prefer if this function returned the `ret`. For example failures in `wc_ecc_init` and `wc_EccPrivateKeyDecode` should return those return codes, not `sss_status_t`.

Reply:
It does this already, if there is no `ret` but an `sss_status_t` error then it will return a `ret` of `WC_HW_E`, like other functions in this file.